e2e/windows/e2e-starter.sh

```diff
@@ -53,7 +54,7 @@ if [ "$create_cluster" == "true" ]; then
     clusterCreateStartTime=$(date +%s)
     retval=0
     
-    az aks create -g $RESOURCE_GROUP_NAME -n $CLUSTER_NAME --node-count 1 --generate-ssh-keys --network-plugin azure -ojson || retval=$?
+    az aks create -g $RESOURCE_GROUP_NAME -n $CLUSTER_NAME --node-count 1 --generate-ssh-keys --network-plugin azure --kubernetes-version $KUBERNETES_VERSION -ojson || retval=$?
```

Comment:
Would it still succeed if $KUBERNETES_VERSION is empty for the default test?

Reply:
Default $KUBERNETES_VERSION was set to 1.24.9 by @junjiezhang1997?

Reply:
Removed the specification. The linux cluster version goes with $KUBERNETES_VERSION defaulting to 1.24.9. The kubernetes version of windows deployment node originates from the version in the package url.

For example if using 1.25.2-hotfix for the windows package url, the deployment node's version is ... as seen on the portal.
`kubectl get nodes` returns
... (*pending*)